brain/python/CMakeLists.txt

```diff
@@ -12,17 +13,32 @@ if( NOT Boost_PYTHON${USE_BOOST_PYTHON_VERSION}_LIBRARY OR
 endif()
 
 set(BRAIN_PYTHON_SOURCE_FILES
+  arrayHelpers.cpp
   brain.cpp
+  circuit.cpp
   spikeReportWriter.cpp
   spikeReportReader.cpp
-  spikes.cpp)
+  spikes.cpp
+  synapses.cpp
+  submodules.cpp
+  neuron/module.cpp
+  neuron/morphology.cpp
+)
+if(BBPTESTDATA_FOUND)
```

Comment:
Some magic code creates the _FOUND variable, whereas the target is always there and is more logical as in a superproject setup nothing is really 'found', it's just there.